Simplenote/SPAppDelegate.m

```diff
@@ -653,22 +668,6 @@ - (BOOL)application:(UIApplication *)app openURL:(NSURL *)url options:(NSDiction
         [_simperium save];
         
         [self presentNote:newNote];
-    } else if ([WPAuthHandler isWPAuthenticationUrl: url]) {
-        if (self.simperium.user.authenticated) {
-            // We're already signed in
-            [[NSNotificationCenter defaultCenter] postNotificationName:kSignInErrorNotificationName
```

Comment:
Just making 100% sure we want this one gone?

Reply:
It seems to be a very edge case to me, to try to log in with WP when you already logged in 🤔 
The only thing this code was doing before is triggering a notification that would show an alert, so I think it can safely go away. 
WDYT? :-)